lib/ui/platform_dispatcher.dart

```diff
@@ -826,6 +826,16 @@ class PlatformDispatcher {
     _onTextScaleFactorChangedZone = Zone.current;
   }
 
+  /// Whether briefly displaying the characters as you type in obscured text
+  /// fields is enabled in system settings.
+  ///
+  /// See also:
+  ///
+  ///  * [EditableText.obscureText], which when set to true hides the text in
+  ///    the text field.
+  bool get textShowPassword => _textShowPassword;
```

Comment:
This name doesn't seem to quite describe the actual behaviour here... should it be something like "brieflyShowPassword" maybe?